### ISSUE TITLE
Point distro packages to this repository

### ISSUE THREAD

**Issue by Salamandar** [syncthing-gtk/syncthing-gtk#6]
We should notify major distros of this new repository.
- [ ] Archlinux
  - [ ] AUR (maintained by @Salamandar )
- [ ] Debian (@hrehfeld ?)
- [ ] Fedora
- [ ] Ubuntu
  - [ ] Mint (is ubuntu enough ?)

**Comment by hrehfeld** [comment]
(I'm not the Debian maintainer, I'm on arch myself)

14 Nov 2022 20:29:37 Salamandar ***@***.***>:

> We should notify major distros of this new repository.
> 
* > Archlinux
  * > AUR (maintained by @Salamandar[https://github.com/Salamandar] )
* > Debian ***@***.***[https://github.com/hrehfeld] ?)
* > Fedora
* > Ubuntu
  * > Mint (is ubuntu enough ?)

**Comment by Rudd-O** [comment]
Let's get the pending changes fixed before we hit the distros, no?

**Comment by Salamandar** [comment]
Yes, of course. I'm just creating this issue for tracking our advancement.
